.github/dependabot.yml

```diff
@@ -3,7 +3,7 @@ updates:
   - package-ecosystem: "bundler"
     directory: "/"
     schedule:
-      interval: "monthly"
+      interval: "weekly"
```

Comment:
🤔 Could we opt-out and keep it as monthly? (cc @KaanOzkan since you changed it in https://github.com/Shopify/rbi/pull/220).